third_party/icu/source/common/cstring.h

```diff
@@ -27,13 +27,10 @@
 #ifndef CSTRING_H
 #define CSTRING_H 1
 
-#if defined(STARBOARD)
-#include "starboard/client_porting/poem/string_poem.h"
-#endif  // defined(STARBOARD)
 #include "unicode/utypes.h"
 #include "cmemory.h"
-#if !defined(STARBOARD)
 #include <string.h>
+#if !defined(STARBOARD)
```

Comment:
For me the Linux build succeeds without this preprocessor conditional.
Do you expect `stdlib.h` or `ctype.h` is a problem?